src/ralph/back_office/models.py

```diff
@@ -41,7 +47,92 @@ class Meta:
 
     @property
     def country_code(self):
-        return 'PL'
+        if self.owner:
+            iso2 = Country.name_from_id(int(self.owner.country)).upper()
+            return iso2_to_iso3(iso2)
+        return 'POL'
 
     def __str__(self):
         return 'BackOfficeAsset: {}'.format(self.hostname)
+
+    @transition_action
+    def assign_user(self, **kwargs):
+        self.user = get_user_model().objects.get(pk=int(kwargs['user']))
```

Comment:
it should be enough to only assign user id here (we don't need to get user instance here and db will validate foreign keys on it's own)